templates/repo/sub_menu.tmpl

```diff
@@ -14,7 +14,7 @@
 						<a href="{{.RepoLink}}/tags">{{svg "octicon-tag"}} <b>{{.NumTags}}</b> {{.locale.TrN .NumTags "repo.tag" "repo.tags"}}</a>
 					</div>
 				{{end}}
-				<div class="item">
+				<div class="item" {{if not (eq .Repository.LFSSize 0)}}data-tooltip-content="{{.Repository.SizeDetailsString}}"{{end}}>
```

Comment:
```suggestion
				<div class="item" {{if not (eq .Repository.Size 0)}}data-tooltip-content="{{.Repository.SizeDetailsString}}"{{end}}>
```

Reply:
@lunny we were thinking tooltip should not be shown in case there is nothing to show i.e. LFS Size = 0. 
I see you are suggesting to always show the tooltip every time size of repository is non-zero. I suppose there is no harm it doing it this way as well. @a1012112796 what do you think? Or you are in for not showing tooltip if lfs is zero?

Reply:
@lunny one other notable point - there is no possibility that `repo.Size == 0`, even empty unitialized repository is 23 KB. 

Anyhow the proposed changes meen that tool-tip would always show. 

@a1012112796 did the PR https://github.com/a1012112796/gitea/pull/10 in case you want it this way.

Reply:
So why it's LFSSize here?

Reply:
It was LFSSize so that tooltip would only be shown if the lfs size is non-zero. There is no need to see tooltip otherwise. If lfssize =0 then the displayed repository size is only git size. So basically only if lfs objects present then tooltip would show the split of size.